.gitignore

```diff
@@ -1,7 +1,31 @@
-# Don't commit
+# Logs
+logs
+*.log
 
-# Installed Node Modules. Commit the package.json instead
-node_modules/
+# Runtime data
+pids
+*.pid
+*.seed
 
-# Bot Config: Contains highly sensitive data, NEVER Commit
-newxkitbot.conf.js
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
```

Comment:
It is GitHub's auto-generated .gitignore, yes. I just added the config/local and storage subfolder